<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StereoscopicConfiguration</class>
 <widget class="QWidget" name="StereoscopicConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>472</height>
   </rect>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2"/>
  </widget>
  <widget class="QTreeView" name="treeView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>631</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblTreeView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Tree View</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
